pkg/github/github.go

```diff
@@ -121,9 +124,20 @@ func getAvailableReleases(
 	}
 
 	// Convert ReleaseInfo array to an array of just the version strings
-	releaseVersions := make([]string, len(releases))
-	for index, release := range releases {
-		releaseVersions[index] = release.TagName
+	releaseVersions := make([]string, 0)
+	for _, release := range releases {
+		versionStr := strings.TrimPrefix(release.Name, "v")
+		_, err := semver.NewVersion(versionStr)
+
+		if err != nil {
+			// Skip adding versions that don't follow semver
+			continue
```

Comment:
We should show warnings or at least some debug info for these as it may indicate a problem in upstream repo

Reply:
I'd want to test first and see how common non-semver tags are. if there are a lot, that could get really verbose

Reply:
added some warnings.